Makefile

```diff
@@ -23,11 +24,11 @@ test:
 
 .PHONY: clean
 clean:
-	-rm -rf dist
+	-rm -rf packages/*/dist
 
 .PHONY: build
 build:
-	$(DOCKER_RUN) npm pack
+	$(DOCKER_RUN) cd packages/skygear && npm pack
```

Comment:
This line expands to the following when `WITH_DOCKER=1`
```
docker run --rm -i -v `pwd`:/usr/src/app -w /usr/src/app skygeario/skygear-nodedev cd packages/skygear && npm pack
```

That means `cd` is treated as a command but it is actually a shell builtin function. Wrap it inside `sh -c "cd ..."` like above.